src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -56,7 +57,7 @@ HttpStatusCode mongoSubscribeContext
   const std::vector<std::string>&      servicePathV
 )
 {
-    std::string        servicePath           = (servicePathV.size() == 0)? "" : servicePathV[0];    
+    std::string        servicePath           = (servicePathV.size() == 0)? DEFAULT_SERVICE_PATH : servicePathV[0];
```

Comment:
Not really part of this PR, just a question ...
We have TWO default service paths,   "/" and "/#", I imagine mongoSubscribeContext uses "/#" as default, as it is some kind of query  ...
However, the define 'DEFAULT_SERVICE_PATH' doesn't really indicate which of the two is in question ...